internal/cmd/cmd_bad_conf_test.go

```diff
@@ -22,6 +22,13 @@ environments:
   gwenv: test
 - prefix: exodus
   gwenv: test2
+`},
+		{"invalid prefix stripped",
```

Comment:
I found an odd bug relating to this test. This commit shouldn't be merged until I've identified the source of and fixed the bug.

I'll mention it here to see if it rings any bells:

If I try to add another `tests` struct here, the exodus-rsync.conf tempfile for the new test seems to be missing (this goes for any invalid config file, not one only containing the "strip" key):
```
[crungeho@localhost exodus-rsync]$ git diff
diff --git a/internal/cmd/cmd_bad_conf_test.go b/internal/cmd/cmd_bad_conf_test.go
index e2c8f18..e47efe8 100644
--- a/internal/cmd/cmd_bad_conf_test.go
+++ b/internal/cmd/cmd_bad_conf_test.go
@@ -29,6 +29,15 @@ strip: invalid
 environments:
 - prefix: exodus
   gwenv: test
+`},
+                {"invalid EnvConfig strip",
+                        `
+environments:
+- prefix: exodus
+  gwenv: test
+  strip: nope
+- prefix: exodus2
+  gwenv: test2
 `},
```

```
[crungeho@localhost exodus-rsync]$ make check
go generate ./...
go test -coverprofile=coverage.out -coverpkg=./... ./...
ok  	github.com/release-engineering/exodus-rsync/cmd/exodus-rsync	0.014s	coverage: 1.4% of statements in ./...
ok  	github.com/release-engineering/exodus-rsync/internal/args	0.023s	coverage: 2.6% of statements in ./...
   • loading config            path=exodus-rsync.conf
   ⨯ can't initialize exodus-gw client error=client error
   ⨯ can't load config         error=can't parse exodus-rsync.conf: yaml: line 1: did not find expected ',' or ']'
   ⨯ can't load config         error=duplicate environment definitions for 'exodus'
   ⨯ can't load config         error=cannot strip 'invalid' prefix from 'exodus'
rsync: [sender] link_stat "/tmp/TestMainBadConf_invalid_EnvConfig_strip2877887328/001/src" failed: No such file or directory (2)

sent 17 bytes  received 12 bytes  58.00 bytes/sec
total size is 0  speedup is 0.00
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1330) [sender=3.2.3]
```

Reply:
For me, just indenting that final 

> `},

(line 32) was enough to fix the issue

Reply:
strange 'cause it wasn't a problem before

Reply:
It may also be worth noting that the test run here in github don't seem to have a problem with the formatting.

Reply:
Ah, thanks! I tried to indent line 32 using my usual method of indentation (spaces) and it didn't fix it for me. Re-formatting the indentations to use tabs fixed the issue. I'm going to have to get used to using tabs, I guess. Gofmt doesn't fix that type of thing?